packages/template-cli/templates/src/ssr/server/index.ts

```diff
@@ -21,14 +21,14 @@ export default app
     .then(() => {
         const server = express()
         server.use(helmetGuard)
-        if (conf.APPINSIGHTS_KEY) {
+        server.use(httpLogger)
+        if (!process.env.CI) {
             appInsights
-                .setup(conf.APPINSIGHTS_KEY)
+                .setup()
                 .setAutoCollectConsole(true)
                 .start()
         }
-        server.use(httpLogger)
```

Comment:
We need thiss?

Reply:
just moved above app insights invocation